src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -2,8 +2,10 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.Serialization.Formatters</AssemblyName>
     <RootNamespace>System.Runtime.Serialization.Formatters</RootNamespace>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```

Comment:
I'm not sure this is worth it, IL linker will do the same job with code you already wrote

Reply:
The split is to throw different exception for the unconditionally disabled variant: https://github.com/dotnet/runtime/pull/38963#issuecomment-658478752

I am not sure whether the different exception is worth the complexity for the unconditionally disabled variant. But if we do want the unconditionally disabled variant to be different, we can go all the way and make the whole type disabled: https://github.com/dotnet/runtime/pull/38963#discussion_r454982675